test/JDBC/input/SP_TABLES-dep-vu-prepare.sql

```diff
@@ -0,0 +1,32 @@
+USE master;
```

Comment:
nit: redundant, as we run any test based on the databaseName set in config.txt in JDBC test framework which is master by default.